.github/workflows/ci.yaml

```diff
@@ -145,22 +145,27 @@ jobs:
 
       # Publish a syncer image to the local registry
       - uses: imjasonh/setup-ko@v0.4
-      - run: KO_DOCKER_REPO=localhost:5000 ko publish ./cmd/syncer --insecure-registry -t $(git rev-parse --short "$GITHUB_SHA") > /tmp/syncer-image
+      - run: KO_DOCKER_REPO=${KIND_REGISTRY} ko publish ./cmd/syncer --insecure-registry -t $(git rev-parse --short "$GITHUB_SHA") > /tmp/syncer-image
```

Comment:
The kind action doesn't use localhost by default. It sets up this env var we can use.